_sass/site/blog/post/_post-images.scss

```diff
@@ -5,6 +5,10 @@ img {
   margin-right: auto;
 }
 
+.img-left {
```

Comment:
Przyznam, że niezbyt mi się to podoba. IMO lepiej by wyglądało wyśrodkowane z opisem w stylu "Wynik działania czegoś tam"